tests/views_integration/Tribe/Events/Views/V2/Partials/Components/Top_Bar/NavTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\Components\Top_Bar;
+
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class NavTest extends HtmlPartialTestCase
+{
+
+	protected $partial_path = 'components/top-bar/nav';
+
+	/**
+	 * Test render with all links
+	 */
+	public function test_render_with_all_links() {
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'prev_url'  => '#',
+			'next_url'  => '#',
+		] ) );
+	}
+
+	/**
+	 * Test render without prev url
+	 */
+	public function test_render_without_prev_url() {
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'next_url'  => '#',
+		] ) );
+	}
+
+	/**
+	 * Test render without next url
+	 */
+	public function test_render_without_next_url() {
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'prev_url'  => '#',
```

Comment:
I'd rather user hard-coded values here, this would be easier to spot in the snapshot if something changes.